modules/local/prokka.nf

```diff
@@ -0,0 +1,50 @@
+process PROKKA {
```

Comment:
same comment about wrapping into folder. I see prokka exists in nf-core 
https://github.com/nf-core/modules/blob/master/modules/nf-core/prokka/main.nf
You can maybe add that issue fix there.. not sure, just noticed it exists in nf-core